ddtrace/tracer.py

```diff
@@ -518,8 +540,8 @@ def start_span(
             ):
                 span._set_str_tag(VERSION_KEY, config.version)
 
-        # add it to the current context
-        context._add_span(span)
+        if activate:
+            self.context_provider.activate(span)
```

Comment:
What was the purpose of allowing spans to not activate but be created again?

Reply:
It allows you to do manual span parenting and not interfere with the automated context management.

One example is in twisted where you might start a span for a deferred but you don't want to activate it because the deferred might not be executed immediately (and don't want new spans to inherit from it)

Reply:
Got it, thanks for the clarification :smile could we add this example in the PR description perhaps?